keyboards/yellowbrickroad/info.json

```diff
@@ -0,0 +1,215 @@
+{
+    "keyboard_name": "yellowbrickroad",
+    "manufacturer": "3araht",
+    "url": "https://github.com/3araht/yellowbrickroad",
+    "maintainer": "3araht",
+    "bootloader": "rp2040",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "encoder": true,
+        "bootmagic": false,
+        "console": false,
+        "mousekey": false,
```

Comment:
```suggestion
        "bootmagic": true,
        "console": false,
        "mousekey": true,
```

Reply:
The bootmagic, console and mousekey are not used in this keyboard so I'll remove these rows.